doc/start/quick-ceph-deploy.rst

```diff
@@ -38,15 +38,12 @@ Create a Cluster
 ================
 
 If at any point you run into trouble and you want to start over, execute
-the following to purge the configuration::
+the following to purge the Ceph package, and erase all its data and configuration::
```

Comment:
Yes , it should be 'packages'.
def uninstall(distro, purge=False):
    packages = [
        'ceph',
        'ceph-release',
        'ceph-common',
        'ceph-radosgw',
        ]
Thank you!